packages/react/src/generators/library/library.ts

```diff
@@ -233,7 +234,10 @@ function addProject(host: Tree, options: NormalizedSchema) {
       executor: '@nrwl/web:rollup',
       outputs: ['{options.outputPath}'],
       options: {
-        outputPath: `dist/${libsDir}/${options.projectDirectory}`,
+        outputPath:
+          libsDir !== '.'
+            ? `dist/${libsDir}/${options.projectDirectory}`
```

Comment:
Can this re-use the extractLayoutDirectory function instead of passing the libsDir down?